shaman/util.py

```diff
@@ -55,3 +55,54 @@ def is_node_healthy(node):
     node.down_count = 0
     models.commit()
     return True
+
+
+def parse_distro_release(identifier):
+    """
+    Back and forth translation for a release identifier, falling back to
+    ``None`` when an identifier has no existing mapping.
+
+    For CentOS, an identifier of '7' will return: None, '7'.
+
+    For an identifier of 'xenial', will return: 'xenial', '16.04'
+
+    returns: 2 item tuple (codename, version)
+    """
+    version_map = {
```

Comment:
I can add Debian, but I am hesitant to add anything else (rpm-based distros mainly)